src/app/pages/splash/containers/splash-page.component.ts

```diff
@@ -101,10 +108,11 @@ export class SplashPageComponent {
         ) {
           this.status = 'Updating notifications...'
           console.log('[SPLASH] Scheduling Notifications.')
-          this.firebaseAnalytics.logEvent('notification_rescheduled', {})
-          return this.notificationService.setNextXNotifications(
-            DefaultNumberOfNotificationsToSchedule
-          )
+          return this.notificationService
+            .setNextXNotifications(DefaultNumberOfNotificationsToSchedule)
+            .then(() =>
+              this.firebaseAnalytics.logEvent('notification_rescheduled', {})
```

Comment:
I think this is not a `rescheduled` event as the notifications are not cancelled first. This should be name `notification_scheduled`